.changeset/three-timers-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/elements": minor
+---
+
+Adds `restricted-access` Step for invite-only applications
```

Comment:
I don't like we call it `invite-only` applications
Maybe we should change to:
Adds `restricted-access` Step for `restricted` sing-up mode ?

Reply:
Sounds good